packages/components/form-control/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@spark-ui/form-control",
+  "version": "0.0.0",
```

Comment:
Maybe for `alpha` or `beta` components we should use version `0` so users can now that the API is unstable and could change (use on their own risk). This way we can avoid huge PRs